cmd/envoy-gateway/main.go

```diff
@@ -0,0 +1,29 @@
+// Copyright The Envoy Project Authors
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// 	http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/envoyproxy/gateway/internal/cmd"
+)
+
+func main() {
+	if err := cmd.GetRootCommand().Execute(); err != nil {
+		fmt.Fprintln(os.Stderr, err)
```

Comment:
lets decide on a logger and/or logging framework either here or via a GH issue
I'd recommend using a generic logging framework thats the project is based on such as https://github.com/tetratelabs/log#common-logging-infrastructure which allows us to pick any logging implementation and swap it out in the future
cc @basvanbeek

Reply:
Yep, the choice of logging framework hadn't been resolved in the open doc so I didn't want to touch it yet, I'd suggest deferring that to a future PR.

Reply:
I created https://github.com/envoyproxy/gateway/issues/76 to track the logging framework requirement. I suggest that #76 does not block this PR.